From: Jeff Gage <jgage@wikimedia.org>
Date: Tue, 9 Sep 2014 04:40:42 -0400
Subject: downgrade-redis

---
 Makefile |    2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 5724928..a1c21e0 100644
--- a/Makefile
+++ b/Makefile
@@ -5,9 +5,7 @@ daemonize yes
 port 6479
 pidfile work/redis1-6479.pid
 logfile work/redis1-6479.log
-save ""
 appendonly no
-client-output-buffer-limit pubsub 256k 128k 5
 endef
 
 export REDIS1_CONF
-- 
